TestCntlrApp/src/enbApp/pcap/bpf.h

```diff
@@ -800,8 +800,8 @@ struct bpf_program {
 #define DLT_RAIF1		198
 
 /*
- * IPMB packet for IPMI, beginning with the I2C slave address, followed
- * by the netFn and LUN, etc..  Requested by Chanthy Toeung
+ * IPMB packet for IPMI, beginning with the I2C passive/secondary address,
```

Comment:
see also https://docs.zephyrproject.org/3.1.0/hardware/peripherals/i2c.html, `I2C slave` is official language from specifilation and just changing it is at least for zephyr not a solution.

> Zephyr recognizes the need to change the terms “master” and “slave” used in the current [I2C Specification](https://docs.zephyrproject.org/3.1.0/hardware/peripherals/i2c-specification). This will be done when the conditions identified in [Rule A.2: Inclusive Language](https://docs.zephyrproject.org/3.1.0/contribute/coding_guidelines/index.html#coding-guideline-inclusive-language) have been met. Existing documentation, data structures, functions, and value symbols in code are likely to change at that point.